source/index.html.md

```diff
@@ -857,18 +2159,215 @@ var options = {
 	filters: {
 		$title: {
 			$contains: 'adventure'
+		},
+		$timestampUpdated: {
```

Comment:
Yup, this is great! I think this shows all the different variations you can use for filters.

"pages" was confusing at first glance, as it's similar to the `page` option for record paging. But this makes sense, just took a second to realize what was going.